gcovr/configuration.py

```diff
@@ -38,6 +38,18 @@ def check_percentage(value):
     return x
 
 
+def check_inputfile(value):
+    r"""
+    Check that the input file is present. Return the full path.
+    """
+
+    if not os.path.isfile(value):
+        raise ArgumentTypeError(
+            "Element {value} is not a file.".format(value=value))
```

Comment:
The word “element” might be confusing. I'd rephrase the message like this to guide the user in the correct direction:
```suggestion
            "Should be a file that already exists: {value!r}".format(value=value))
```

Showing the `!r` repr is useful if the filename contains trailing spaces or nonprintable characters

Reply:
Ok.